src/main/java/it/gov/pagopa/arc/dto/TransactionDTO.java

```diff
@@ -0,0 +1,27 @@
+package it.gov.pagopa.arc.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.ZonedDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TransactionDTO {
+
+    private String transactionId;
+    private String payeeTaxCode;
+    private String amount;
+    private ZonedDateTime transactionDate;
```

Comment:
check if it's required to use @DateTimeFormat as done in https://github.com/pagopa/idpay-transactions/blob/main/src/main/java/it/gov/pagopa/idpay/transactions/dto/MerchantTransactionDTO.java#L21